docs/src/pages/blog/anywidget-02.md

```diff
@@ -215,6 +215,40 @@ export default { render };
 > Note: The above front-end code requires transformation with tool like
 > `esbuild` to allow for the special JSX syntax (e.g., `<App />`).
 
+Another common scenario requiring proper cleanup is when using
+`requestAnimationFrame` to continually redraw into a canvas:
+
+```javascript
+class MyCustomRenderer {
+	constructor() {
+		this.render = this.render.bind(this);
+		this.redrawRequest = requestAnimationFrame(this.render);
+		this.canvasEl = document.createElement("canvas");
+	}
+
+	render() {
+		this.redrawRequest = requestAnimationFrame(this.render);
+
+		// ...custom rendering goes here...
+	}
+}
+
+export default {
+	render({ model, el }) {
+		const renderer = new MyCustomRenderer();
+
+		el.appendChild(renderer.canvasEl);
+
+		return () => {
+			cancelAnimationFrame(renderer.redrawRequest);
+		};
+	},
+};
```

Comment:
Love this! Thanks for the contribution. I think for now, this makes total sense to put in this blog post... but one of these days I'd like to pull out useful nuggets from these for some part of the dedicated documentation.

For this example, the only thing I'm wondering is if we could inline some of this within `render` rather than introducing new syntax like a `class`. It's mostly what would be familiar to an end user. I'm guessing if you're using 	requestAnimationFrame`, a class would probably be familiar .... so maybe i'm over thinking it.

```js
export default {
	render({ model, el }) {
		const canvasEl = document.createElement("canvas");
		let requestId = requestAnimationFrame(step);
		
		function step() {
			/* ...custom rendering goes here... */
			requestId = requestAnimationFrame(step);
		}
		el.appendChild(canvasEl);

		return () => {
			cancelAnimationFrame(requestId);
		};
	},
};
```

Stealing the variable names from [`MDN`](https://developer.mozilla.org/en-US/docs/Web/API/Window/cancelAnimationFrame).

Reply:
Yes, this looks much better, more self-contained! I'll make the change.